hudi-client/hudi-client-common/src/main/java/org/apache/hudi/execution/HoodieLazyInsertIterable.java

```diff
@@ -42,6 +43,8 @@
 public abstract class HoodieLazyInsertIterable<T extends HoodieRecordPayload>
     extends LazyIterableIterator<HoodieRecord<T>, List<WriteStatus>> {
 
+  private static final Properties EMPTY_PROPERTIES = new Properties();
```

Comment:
Why don't we actually use `Option.empty`? If you really want to do this. can you move this to `CollectionUtils`

Reply:
The HoodieRecordPayload#getInsertValue required a `Properties` parameter, so here we must pass a properties object. Move the EMPTY_PROPERTIES to `CollectionUtils` make sense to me !